app/services/exports/case_log_export_constants.rb

```diff
@@ -0,0 +1,138 @@
+module Exports::CaseLogExportConstants
+  MAX_XML_RECORDS = 10_000
+  LOG_ID_OFFSET = 300_000_000_000
+
+  QUARTERS = {
+    0 => "jan_mar",
+    1 => "apr_jun",
+    2 => "jul_sep",
+    3 => "oct_dec",
+  }.freeze
+
+  EXPORT_FIELDS = Set[
+    "armedforces",
+    "beds",
+    "benefits",
+    "brent",
+    "builtype",
+    "cap",
+    "cbl",
+    "chcharge",
+    "chr",
+    "cligrp1",
+    "cligrp2",
+    "createddate", # New metadata coming from our system
```

Comment:
If that's something that we define, maybe `created_date` would be better?

Reply:
I would agree with you but the issue is that the counterpart is `uploadddate` which is already defined. CDS seems to avoid using underscores in general.

Reply:
Hm, I might be misreading the comment, so this metadata is already expected in the export?

Reply:
No no, it is not. I am following the existing convention. If we don't, we will end up with new fields using underscores and old ones concatenated.

I am not saying I find the convention great, but I think staying consistent is more important.

Reply:
But you might argue there are already fields with underscores, so happy to go with what the team prefers tbh.

Reply:
Tricky one, I would probably also follow the trend here, which looks like predominately there are composite names without underscores.